.github/ISSUE_TEMPLATE/BUG_REPORT.md

```diff
@@ -22,24 +27,14 @@ about: Report a bug or unexpected behavior.
 
 <!--- Optional: suggest a fix or reason for the bug if you have a solution -->
 
-## Steps to Reproduce
-
-<!--- Provide clear steps to reproduce. Include code samples if relevant -->
-
-1.
-2.
-3.
-4.
-
-## Context
-
-<!--- How has this bug affected you? What were you trying to accomplish? -->
-
 ## Your Environment
 
 <!--- Include as many relevant details about the environment you experienced the bug in -->
 
-- Version used:
-- Target codebase stack (e.g. Ruby):
 - Operating System and version:
```

Comment:
Maybe we should output the OS and such as part of the version (or maybe have a command to display this kind of info).